client/src/components/Layout/Sider/Sider.tsx

```diff
@@ -83,7 +91,11 @@ export default function Sider({ className, isDrawer }: SiderProps) {
       <div className={s.title}>
         <SiderTitle />
       </div>
-      <SiderSearch onArtistClick={goToArtist} onTrackClick={goToTrack} />
+      <SiderSearch
+        onArtistClick={goToArtist}
+        onTrackClick={goToTrack}
+        onAlbumClick={goToAlbum}
```

Comment:
Nitpick, but would be nicer to have onTrackClick, onAlbumClick, onArtistClick in this order, from small container to bigger